### DISCUSSION TITLE
MCU Channel Color

### DISCUSSION THREAD

**Discussion by mat-tgn** [tttapa/Control-Surface#1075] [Q&A]
Hi, i can get my mixing station channel name from MCU::LCD , but i cannot find a way to get the mixer channel color

**Comment by tttapa** [comment]
If the color information is sent to the Arduino, you should be able to see it in the output of the [Mackie-Control-Universal-Reverse-Engineering.ino](https://tttapa.github.io/Control-Surface-doc/Doxygen/d3/d85/Mackie-Control-Universal-Reverse-Engineering_8ino-example.html) example.

**Reply by mat-tgn** [reply]
thank you i will check

**Comment by mat-tgn** [comment]
I receive this messages changing channel color: 

System Exclusive message: [15] F0 00 00 66 15 72 01 07 07 07 07 07 07 07 F7 on cable 1
System Exclusive message: [15] F0 00 00 66 15 72 03 07 07 07 07 07 07 07 F7 on cable 1
System Exclusive message: [15] F0 00 00 66 15 72 02 07 07 07 07 07 07 07 F7 on cable 1
System Exclusive message: [15] F0 00 00 66 15 72 04 07 07 07 07 07 07 07 F7 on cable 1
System Exclusive message: [15] F0 00 00 66 15 72 05 07 07 07 07 07 07 07 F7 on cable 1

Can you tell me what is the better way to manage it ? i want to change rgb led color according to selected color ( 1 - 5 )

thank you in advice

**Reply by tttapa** [reply]
Have a look at the [SysEx-Send-Receive](https://tttapa.github.io/Control-Surface-doc/Doxygen/d2/d93/SysEx-Send-Receive_8ino-example.html) example and the [MIDI tutorial](https://tttapa.github.io/Control-Surface-doc/Doxygen/d3/df7/midi-tutorial.html#midi_md-receiving).  
In the callback, check the length of the message and the contents, and if it matches one of the messages you expect, you access the color using `sysex.data[6]`.
